### PR TITLE
slim6

### PR DESCRIPTION
**Pull request by anbuchelva**

### CODE REVIEW

**Comment by regalstreak** [comment]
Done. Will be up in a few hours.

**Comment by msfjarvis** [comment]
Not coming, the URL is malformed, needs the scheme (https://) also. I will
not edit this locally, the people need to know how it works.

File: compress.bash [hunk 1]
```diff
@@ -1,6 +1,6 @@
-name=AOSP
-manifest=https://android.googlesource.com/platform/manifest -b android-4.0.1_r1
-branch=android-6.0.1_r55
+name=slim6
+manifest=github.com/SlimRoms/platform_manifest.git
```
Comment (msfjarvis) [inline compress.bash:2]: `https://` is also needed, you could have seen that in the existing URL @anbu

**Comment by regalstreak** [comment]
lol i didnt even check

**Comment by regalstreak** [comment]
yes @anbuchelva you need to submit another pr

**Comment by msfjarvis** [comment]
I'll handle this now, you chill with XOS :)

**Comment by anbuchelva** [comment]
@regalstreak @MSF-Jarvis sorry for the mistake.. anyway i've downloaded CM source and used it as reference to build slimroms.  thank you!!

**Comment by msfjarvis** [comment]
@regalstreak, did the guy make any sense?

**Comment by anbuchelva** [comment]
hope this makes sense..
repo init --reference=~/android/cm13 -u git://github.com/SlimRoms/platform_manifest.git -b mm6.0

**Comment by msfjarvis** [comment]
Yeah that one worked, I'd have added something like this to skadoosh if
we'd have been short on bandwidth. Sadly we are not.

**Comment by regalstreak** [comment]
nice, nice.

**Pull request merged**